docs/source/tutorial/visualise_pipeline.md

```diff
@@ -233,3 +235,80 @@ You can view the larger visualisation of the chart by clicking the 'Expand Plotl
 
 
 ![](../meta/images/pipeline_visualisation_plotly_expand.png)
+
+
+### Visualise Matplotlib charts in Kedro-Viz
+
+[Matplotlib](https://matplotlib.org/) is a Python library for creating static, animated, and interactive visualizations. Integrating Matplotlib into Kedro-Viz allows you to output your charts as part of your pipeline visualisation.
```

Comment:
```suggestion
[Matplotlib](https://matplotlib.org/) is a Python library for creating static, animated, and interactive visualisations. Integrating Matplotlib into Kedro-Viz allows you to output your charts as part of your pipeline visualisation.
```

Reply:
not sure I am thinking maybe remove animated, and interactive since Kedro-viz only supports static. I know we add it somewhere in the bottom but it still might confuse.

Reply:
Maybe we can put the note on Kedro-viz supports only static charts right after this.